services/espnet-tts/Dockerfile

```diff
@@ -10,18 +12,43 @@ USER python
 ENV PATH="/home/python/.local/bin:${PATH}"
 
 RUN pip install --upgrade pip
-COPY /services/espnet-tts/requirements.txt .
+COPY ./requirements.txt .
+# COPY /services/espnet-tts/requirements.txt .
```

Comment:
You cannot actually use relative paths here for how the build actually works. In order to access the schemas, your working directory for builds **must** be the root of the repository. This is why you needed to copy the schema files below (needlessly).